README.md

```diff
@@ -149,7 +149,8 @@ cookie duration (defined in hours).
               },
               cacheName: 'mylang', // default value is 'lang'.
               cacheMechanism: 'Cookie', // default value is 'LocalStorage'.
-              cookieExpiry: 1, // default value is 720, a month.
+              cookieExpiry: 1, // default value is 720, a month, or use 'COOKIE_EXPIRE_SESSION' for session expiry.
```

Comment:
Would something like this make sense?
`// default value is 720, a month. Set to a negative value and the cookie becomes a session cookie.`

Reply:
Yeah that sounds sensible to me! See most recent commit.